docs/Customizing Look and Functionality/Customizing the Keyboard/index.md

```diff
@@ -33,8 +33,8 @@ sdlManager.screenManager.keyboardConfiguration = keyboardConfig
 @![android, javaSE, javaEE]
 ```java
 KeyboardProperties keyboardProperties = new KeyboardProperties();
```

Comment:
I suggested doing the initialization and setting the properties in one line. Fore example:
```
Alert alert = new Alert()
        .setAlertText1("Hello")
        .setAlertText2("Hi there!")
        .setAlertIcon(sdlArtwork.getImageRPC());
```